understanding/focus-order.html

```diff
@@ -58,205 +56,151 @@
+                  				達成基準 2.4.3:</span>フォーカス順序 (レベル A)
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>If a <a href="#dfn-web-page">Web page</a> can be <a href="#dfn-navigated-sequentially">navigated sequentially</a> and the navigation sequences affect meaning or operation, focusable components receive
-                     focus in an order that preserves meaning and operability.
+                  <p><a href="#dfn-web-page">ウェブページ</a>が<a href="#dfn-navigated-sequentially">順を追ってナビゲート</a>できて、そのナビゲーション順が意味又は操作に影響を及ぼす場合、フォーカス可能なコンポーネントは、意味及び操作性を損なわない順序でフォーカスを受け取る。
                      
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Keyboard users navigate content in a correct order.</dd>
+                  <dt>目標</dt>
+                  <dd>キーボードの利用者が、適切な順序でコンテンツをナビゲートできる。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Elements receive focus in an order that preserves meaning.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>各要素が、意味を保った順序でフォーカスを受け取るようにする。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>Navigating a site or application with only the keyboard will make sense.</dd> 
+                  <dt>なぜそれが重要か</dt>
+                  <dd>キーボードだけでサイト又はアプリケーションをナビゲートしても意味が通る。</dd> 
                   
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The intent of this Success Criterion is to ensure that when users navigate sequentially
-                  through content, they encounter information in an order that is consistent with the
-                  meaning of the content and can be operated from the keyboard. This reduces confusion
-                  by letting users form a consistent mental model of the content. There may be different
-                  orders that reflect logical relationships in the content. For example, moving through
-                  components in a table one row at a time or one column at a time both reflect the logical
-                  relationships in the content. Either order may satisfy this Success Criterion.
+               <h2>意図</h2>
+               <p>この達成基準の目的は、利用者がコンテンツ内を順を追ってナビゲートするときに、コンテンツの意味と一致し、キーボードから操作可能な順序で、情報に出会うことを保証することである。このことにより、利用者のコンテンツに対するメンタルモデルを一貫したものとし、利用者が困惑する可能性を引き下げる。コンテンツの論理的な関係性を反映した順序が複数存在することもある。例えば、テーブル内のコンポーネント群の中を 1 行ずつ、又は 1 列ずつ移動することは、どちらもコンテンツの論理的な関係を反映している。どちらの順序もこの達成基準を満たすことができる 。
```

Comment:
末尾の句点の前に変なスペースがありますね。

```suggestion
               <p>この達成基準の目的は、利用者がコンテンツ内を順を追ってナビゲートするときに、コンテンツの意味と一致し、キーボードから操作可能な順序で、情報に出会うことを保証することである。このことにより、利用者のコンテンツに対するメンタルモデルを一貫したものとし、利用者が困惑する可能性を引き下げる。コンテンツの論理的な関係性を反映した順序が複数存在することもある。例えば、テーブル内のコンポーネント群の中を 1 行ずつ、又は 1 列ずつ移動することは、どちらもコンテンツの論理的な関係を反映している。どちらの順序もこの達成基準を満たすことができる。
```